pom.xml

```diff
@@ -2979,7 +2983,7 @@
       <properties>
         <hadoop.version>3.2.0</hadoop.version>
         <curator.version>2.13.0</curator.version>
-        <zookeeper.version>3.4.13</zookeeper.version>
+        <zookeeper.version>3.4.14</zookeeper.version>
```

Comment:
I think you can just remove this if the profile does not change the value.

Reply:
OK. I will remove this line.